src/vcf/validate_optional_policy.cpp

```diff
@@ -146,7 +151,8 @@ namespace ebi
             state.add_well_defined_meta("contig", current_chromosome);
         } else {
             state.add_bad_defined_meta("contig", current_chromosome);
-            throw ParsingWarning("Chromosome/contig '" + current_chromosome + "' is not described in a 'contig' meta description");
+            throw MetaSectionError(state.n_lines,
+                    "Chromosome/contig '" + current_chromosome + "' is not described in a 'contig' meta description");
```

Comment:
If the exception in https://github.com/EBIvariation/vcf-validator/pull/22/files#diff-b7ed7bfe2cd51f747bc5f560a4647ee3R208 comes from checking a relationship between meta and body and throws a BodySectionError, shouldn't this one and the following in this file do the same?